_posts/2015-11-25-drupal-6-to-8.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: "It's nearly 2016, your site is running on Drupal 6, now what? (Part 1)"
+date: 2015-11-25
+author: Chris Russo
+tags: drupal drupal8 drupal-planet 
+summary: What to plan for as Drupal 6 EOL approaches.
+
+---
+
+## The issue at <img src="/assets/img/hand.png" alt="hand">
+
+As most Drupal 6 site owners are aware, after a [prolonged development period](https://www.drupal.org/core/dev-cycle), Drupal 8 was officially released ([8.0.0]((https://www.drupal.org/node/2619030))) last week on November 19th, 2015
+[Dries's birthday](https://en.wikipedia.org/wiki/Dries_Buytaert) with a corresponding many, [many a lively party](https://twitter.com/hashtag/Celebr8D8?src=hash): 
+<img src="/assets/img/celebr8d8.jpg" alt="Drupal 8 celebration #celebr8d8">
+<span class="caption">Like this fancy one in downtown Durham atop the rooftop bar of [The Durham Hotel](http://thedurham.com/).</span> 
+
+Drupal 8.0.0 is a [**BIG DEAL**](http://celebratedrupal8.com/) and generally speaking is great for the community of Drupal site owners and site builders.
+
+However (there's always a but), with the official release of Drupal 8, 
+**support for Drupal 6 will end on** [**February 24th, 2016**](https://www.drupal.org/drupal-6-eol). Given the U.S. holiday 
+season has begun, there is little, productive time remaining to undertake a site upgrade before Drupal 6 End Of Life (EOL). If you are a site owner fortunate enough to have survived a Drupal site
```

Comment:
There's a point at which I have to believe that you do this strictly to bother me. "...there is little productive time remaining to undertake..."